src/wp-includes/class-wp-block-type.php

```diff
@@ -562,6 +580,29 @@ public function set_props( $args ) {
 			}
 		}
 
+		// Stabilize experimental block supports.
```

Comment:
This code intentionally does _not_ unset the experimental version of the supports. This is so that the un-updated JS code in the editor can handle the experimental syntax, and (for now at least) feels slightly safer in terms of backwards compatibility.